docker-compose-reload.yml

```diff
@@ -0,0 +1,19 @@
+version: '3'
+
+services:
+
+  appwrite:
+    container_name: appwrite
+
+  live-reloader:
+    image: apogiatzis/livereloading
+    container_name: livereloader
+    privileged: true
+      environment:
+        - RELOAD_CONTAINER=appwrite
+        - RELOAD_DELAY=20              # seconds
+        - RESTART_TIMEOUT=10         # seconds
+        - RELOAD_DIR=/usr/share/nginx/html/public
```

Comment:
Let's change the target branch for this change from master to `0.7.x`, we had lots of architecture changes between them, and hot-reloading will be much more suitable for 0.7 version.

After that, this are the 3 main libraries we need to listen for changes that will reload the main `appwrite` container:
- ./app:/usr/src/code/app
- ./docs:/usr/src/code/docs
- ./public:/usr/src/code/public
- ./src:/usr/src/code/src

Reading the docs, multiple DIRS can be supplied comma separated.